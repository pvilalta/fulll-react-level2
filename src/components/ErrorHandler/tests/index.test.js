import React from 'react';
import { render, screen } from '@testing-library/react';
import Component from '..';

describe('components | ErrorHandler | index', () => {
  it('should render component', () => {
    const message = 'random message';

    const { container } = render(<Component message={message} />);
    const spanMessage = screen.getByTestId('errorMessage').textContent;

    expect(container).toMatchSnapshot();
    expect(spanMessage).toBe('random message');
  });
});
