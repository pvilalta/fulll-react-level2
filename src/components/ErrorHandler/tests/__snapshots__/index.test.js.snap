// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components | ErrorHandler | index should render component 1`] = `
<div>
  <div
    class="ErrorHandler"
  >
    <span
      data-testid="errorMessage"
    >
      random message
    </span>
  </div>
</div>
`;
